### DISCUSSION TITLE
De-Zod-ification - what brought you here?

### DISCUSSION THREAD

**Discussion by marek-hanzal** [open-circle/valibot#218] [General]
I've got just an idea - I'm curious about the `valibot` users, doesn't matter if production or sandbox ones, what brought them to `valibot` or form the other side, why they've leaved whatever prior solution was (Zod, Yup, ...)?

So if there is somebody using valibot reading this, I would like to hear your voice (...you know, what I mean ;) ).

As a last point I want to say that for the first look and partial project migration to valibot it looks very good and I like quite lightweight types, so I'm not polluted all the times by the miles of Zod's errors (I don't want to say Zod is bad, just this was quite hard for me).

 - Good work, man!

**Comment by fabian-hiller** [comment]
Thank you! 💚
